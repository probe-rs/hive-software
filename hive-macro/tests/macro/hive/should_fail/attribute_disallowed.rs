#[hive_macro::hive(disallowed_attribute = 1)]
mod tests {}

fn main() {}
