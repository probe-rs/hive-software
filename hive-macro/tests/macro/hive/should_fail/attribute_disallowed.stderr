error: This macro does not support attributes
 --> tests/macro/hive/should_fail/attribute_disallowed.rs:1:20
  |
1 | #[hive_macro::hive(disallowed_attribute = 1)]
  |                    ^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = help: Remove any macro attributes
